card_network.py

```diff
@@ -0,0 +1,66 @@
+import torch
```

Comment:
add unittest for this file, note you should add a `fake_data.py` to generate fake env data and test it